channels/1ds-post-js/src/PostChannel.ts

```diff
@@ -306,6 +346,38 @@ export class PostChannel extends BaseTelemetryPlugin implements IChannelControls
                     }
                 }
 
+            }
+
+            function _addEventToQueues(event: IPostTransmissionTelemetryItem, append: boolean) {
+                // If send attempt field is undefined we should set it to 0.
+                if (!event.sendAttempt) {
+                    event.sendAttempt = 0;
+                }
+                // Add default latency
+                if (!event.latency) {
+                    event.latency = EventLatencyValue.Normal;
+                }
+                _cleanEvent(event);
+
+                // // Remove extra AI properties if present
+                // if (event.ext && event.ext[STR_TRACE]) {
+                //     delete (event.ext[STR_TRACE]);
+                // }
+                // if (event.ext && event.ext[STR_USER] && event.ext[STR_USER]["id"]) {
+                //     delete (event.ext[STR_USER]["id"]);
+                // }
+
+                // // v8 performance optimization for iterating over the keys
+                // if (_optimizeObject) {
```

Comment:
We actually, will still need this as it dramatically affects the performance for serializing large modified objects on chromium